__mocks__/jestAPICredentials.js

```diff
@@ -2,6 +2,8 @@ export const jestAPICredentials = {
   BASE_COMMENTS_API:
     'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/',
   COMMENTS_ENDPOINT_ID: 'QhwyHRqxtZN5QJVQP2gT',
+  LIKES_ENDPOINT_API:
```

Comment:
As we talked previously, we were not able to extract credentials data from .env file while using jest. So we add credentials over here for testing. We are going to hide that file when we are done with everything.